lib/experimental/html/class-wp-html-tag-processor.php

```diff
@@ -1331,16 +1331,33 @@ public function seek( $bookmark_name ) {
 	}
 
 	/**
-	 * Sort function to arrange objects with a start property in ascending order.
+	 * Compare two WP_HTML_Text_Replacement objects.
 	 *
 	 * @since 6.2.0
 	 *
-	 * @param object $a First attribute update.
-	 * @param object $b Second attribute update.
+	 * @param WP_HTML_Text_Replacement $a First attribute update.
+	 * @param WP_HTML_Text_Replacement $b Second attribute update.
 	 * @return integer
 	 */
 	private static function sort_start_ascending( $a, $b ) {
-		return $a->start - $b->start;
+		$by_start = $a->start - $b->start;
+		if ( $by_start !== 0 ) {
```

Comment:
```suggestion
		if ( 0 !== $by_start ) {
```
https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#yoda-conditions